kratos/includes/define_python.h

```diff
@@ -35,6 +35,52 @@
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(module,name##_Y) \
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(module,name##_Z)
 
+#ifdef KRATOS_REGISTER_IN_PYTHON_SYMMETRIC_2D_TENSOR_VARIABLE_WITH_COMPONENTS
+#undef KRATOS_REGISTER_IN_PYTHON_SYMMETRIC_2D_TENSOR_VARIABLE_WITH_COMPONENTS
+#endif
+#define KRATOS_REGISTER_IN_PYTHON_SYMMETRIC_2D_TENSOR_VARIABLE_WITH_COMPONENTS(name) \
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(name) \
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(name##_XX) \
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(name##_YY) \
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(name##_XY)
+
+#ifdef KRATOS_REGISTER_IN_PYTHON_SYMMETRIC_3D_TENSOR_VARIABLE_WITH_COMPONENTS
+#undef KRATOS_REGISTER_IN_PYTHON_SYMMETRIC_3D_TENSOR_VARIABLE_WITH_COMPONENTS
+#endif
+#define KRATOS_REGISTER_IN_PYTHON_SYMMETRIC_3D_TENSOR_VARIABLE_WITH_COMPONENTS(name) \
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(name) \
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(name##_XX) \
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(name##_YY) \
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(name##_ZZ) \
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(name##_XY) \
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(name##_YZ) \
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(name##_XZ)
+
+#ifdef KRATOS_REGISTER_IN_PYTHON_2D_TENSOR_VARIABLE_WITH_COMPONENTS
+#undef KRATOS_REGISTER_IN_PYTHON_2D_TENSOR_VARIABLE_WITH_COMPONENTS
+#endif
+#define KRATOS_REGISTER_IN_PYTHON_2D_TENSOR_VARIABLE_WITH_COMPONENTS(name) \
```

Comment:
wouldn't it be better to use the NON_SYMMETRIC keyword to make it very explicit in the macro name?

Reply:
Humm, could be, in any case it is a convection name